docs/src/README.md

```diff
@@ -0,0 +1,38 @@
+# Fuel Indexer
+
+![Fuel Logo](https://miro.medium.com/fit/c/176/176/1*W-1e0zVhcLui4u-ci2e-Fg.png "title")
+
+[![build](https://github.com/FuelLabs/fuel-indexer/actions/workflows/ci.yml/badge.svg)](https://github.com/FuelLabs/fuel-indexer/actions/workflows/ci.yml)
+[![crates.io](https://img.shields.io/crates/v/fuel-indexer?label=latest)](https://crates.io/crates/fuel-indexer)
+[![docs](https://docs.rs/fuel-indexer/badge.svg)](https://docs.rs/fuel-indexer/)
+[![discord](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
+
+Fuel Indexer uses WASM to index transactions and state within a fuel network, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases. For more info on how to get started with Fuel Indexer, [read the book](https://fuellabs.github.io/fuel-indexer/latest/index.html)
+
+## Documentation
+
+### Building From Source
+
+Install `mdbook` and then open a new terminal session in order to run the subsequent commands
+
+```sh
+
+cargo install mdbook
+
+```
```

Comment:
Only need space before opening and after closing, not after opening and before closing.
````suggestion
```sh
cargo install mdbook
```
````

Reply:
Hm yea I see there's a visual difference between the 2. I wonder why markdownlint doesn't complain about this. Will make the change